ledger/controllers/grants.js

```diff
@@ -208,6 +208,18 @@ v1.write = { handler: (runtime) => {
       return reply(boom.badData('promotion already applied to wallet'))
     }
 
+    if (runtime.config.balance) {
+      // invalidate any cached balance
+      await braveHapi.wreck.delete(runtime.config.balance.url + '/v2/wallet/' + paymentId + '/balance',
+        {
+          headers: {
+            authorization: 'Bearer ' + runtime.config.balance.access_token,
+            'content-type': 'application/json'
+          },
+          useProxyP: true
+        })
+    }
+
```

Comment:
i suggest putting this in a `try ... catch` which sentry-logs the failure...

Reply:
good idea :+1: